x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/get_states/v1.ts

```diff
@@ -43,3 +47,32 @@ export const getCspBenchmarkRulesStatesHandler = async (
     );
   }
 };
+
+export const buildMutedRulesFilter = async (
+  encryptedSoClient: ISavedObjectsRepository
+): Promise<QueryDslQueryContainer[]> => {
+  const rulesStates = await getCspBenchmarkRulesStatesHandler(encryptedSoClient);
+
+  const mustNotFilter = [];
+  const mutedRules = Object.fromEntries(
+    Object.entries(rulesStates).filter(([key, value]) => value.muted === true)
+  );
+  for (const key in mutedRules) {
+    if (mutedRules.hasOwnProperty(key)) {
```

Comment:
no need for this check as the object returned from `Object.fromEntries` should be safe from inherited properties